sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -32,6 +31,8 @@ import org.apache.spark.sql.execution.datasources.{CreateTable, DataSource}
 import org.apache.spark.sql.types.StructType
 import org.apache.spark.storage.StorageLevel
 
+import scala.util.Try
```

Comment:
Hi, @httfighter . Could you run the Scala style check first?
```
$ dev/scalastyle
```

Reply:
@dongjoon-hyun Thank you! I have fixed it.